### PR TITLE
feat:Add landing page

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- landing page 
- [code sandbox](https://jkj6qd.csb.app/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,8 +5,40 @@
...
+                <span class="un-txt">" Responsive "</span> web apps.
+            </p>
+            <div>
+                <a href="#">
+                    <button class="primary">Get started</button>
+                </a>
+
+                <a href="#">
+                    <button class="secondary">Github</button>
+                </a>
```
Comment (anupkrbid) [inline index.html:34]: You should not put a button inside an `a` tag. If the purpose is to navigate the user to a different page then use the `a` tag directly or else use a `button`. If you want the `a` tag to look like a button then use the same class to style both `button` and `a` tag.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    font-size: 55px;
+    margin-left: 45px;
+}
+.a{
+    justify-content: flex-end;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.a1{
```
Comment (anupkrbid) [inline main.css:22]: use a better class name that gives it meaning. currently, I'm not able to understand the meaning of a1.

**Review by anupkrbid — COMMENTED** [review]
I've added some comments on the structure. Please check that.
I can see that there is a blank space under the footer of your landing page. Please try to fix that too.

**Review by poojakelkar — APPROVED** [review]
increase little bit font size, otherwise looks good to merge

**Review by Rajitha04 — APPROVED** [review]
Looks good to merge!

**Pull request merged**